### ISSUE TITLE
How to use CSP `nonce` option with Rails 5.2?

### ISSUE THREAD

**Issue by Spone** [gazay/gon#250]
Since 5.2, Rails supports adding a `nonce` to `script` tags (see http://guides.rubyonrails.org/security.html#content-security-policy)

What's the recommended way to use it with gon? I managed to make it work with:

```
<%= javascript_tag nonce: true do -%>
  <%= include_gon(need_tag: false) -%>
<% end %>
```

but I'd rather do it in one line. Is there a way?

**Comment by gazay** [comment]
Hello! There is an option for that https://github.com/gazay/gon/wiki/Options#nonce--foo

**Comment by Spone** [comment]
Great! But I think you could update the documentation, as nonce generation is now included in Rails 5.2.

You no longer need to use `secure_headers` gem, you can just do:

```erb
<%= include_gon nonce: content_security_policy_nonce %>
```
